### ISSUE TITLE
Order placement menu: Adapt units/amounts in footer to new unit system

### ISSUE THREAD

**Issue by twothreenine** [foodcoopsat/foodsoft_hackathon#63]
This is outdated and confusing:
![grafik](https://github.com/foodcoopsat/foodsoft_hackathon/assets/22921479/9ed8d728-aff0-4d12-8b09-240ba65a0213)

Suggestion: (omit tolerance for case potatoes, as there's no tolerance input anyway):
![grafik](https://github.com/foodcoopsat/foodsoft_hackathon/assets/22921479/6309fb83-0571-4445-9056-ba73e125e05f)

----------------

In case of detergent -- old:
![grafik](https://github.com/foodcoopsat/foodsoft_hackathon/assets/22921479/e6485658-63cc-4839-8834-1a150b57bc93)

Suggestion:
![grafik](https://github.com/foodcoopsat/foodsoft_hackathon/assets/22921479/8b4e3c25-add6-4247-b08f-ae4e8f0c109f)

----------------

So, my suggestion is:
1. Rename "Filled units" ("Volle Gebinde") to "Order result" ("Bestellergebnis") - it's the amount that will be ordered from the supplier (in supplier order unit) if the minimum order quantity is reached, so it's not only about filling units anymore.
2. Append the supplier order unit to the _order result_ amount
3. Omit "Units: ..." on the left, since it's included in _order result_
4. Rename "Total units" ("Gesamt-Einheiten") to "Total amount" ("Gesamtmenge"), so that it is consistent with the column titles
5. Omit "Total tolerance" if tolerance input is omitted anyway for the respective article

**Comment by lentschi** [comment]
I agree that this is currently a bit confusing and might require a redesign, but I think we could live with the current fields in a first version and move this to the "Post-merge" milestone. (Then we might be able to make such a redesign with more people involved.)
